guard/src/commands/validate/common.rs

```diff
@@ -483,8 +485,9 @@ pub(super) fn extract_name_info<'a>(
                 .map_or(
                     ("".to_string(), "".to_string()),
                     |msg| match PATH_FROM_MSG.captures(msg) {
-                        Some(cap) => (cap["path"].to_string(), msg.clone()),
-                        None => ("".to_string(), msg.clone()),
+                        Ok(Some(cap)) => (cap["path"].to_string(), msg.clone()),
+                        Ok(None) => ("".to_string(), msg.clone()),
+                        Err(e) => ("".to_string(), e.to_string()),
```

Comment:
if theres an error here is this really what we want to be doing? I feel like it's not.